tests/validation/tests/v3_api/test_websocket.py

```diff
@@ -6,6 +6,10 @@
 from .common import WebsocketLogParse
 from .common import USER_TOKEN
 from .common import create_connection
+from .common import create_project_and_ns
+from .common import random_test_name
+from .common import TEST_IMAGE
```

Comment:
group imports following the PEP8 
 
- standard library imports
- related third party imports
- local application/library specific imports

and alphabetize within those groups (all caps go first).